packages/renderer/src/lib/ingresses-routes/IngressRouteColumnHostPath.svelte

```diff
@@ -0,0 +1,28 @@
+<script lang="ts">
+import Link from '../ui/Link.svelte';
+import type { IngressUI } from './IngressUI';
+import type { RouteUI } from './RouteUI';
+
+export let object: IngressUI | RouteUI;
+</script>
+
+{#if 'rules' in object && object.rules}
+  {#each object.rules as rule}
+    {@const host = rule.host ?? ''}
```

Comment:
Ditto to Backend comment, maybe move logic to util class and store array in IngressUI/RouteUI, keeps UI component simpler and just about display.